packages/roosterjs-content-model/lib/formatHandlers/block/lineHeightFormatHandler.ts

```diff
@@ -6,7 +6,27 @@ import { LineHeightFormat } from '../../publicTypes/format/formatParts/LineHeigh
  */
 export const lineHeightFormatHandler: FormatHandler<LineHeightFormat> = {
     parse: (format, element, context, defaultStyle) => {
-        const lineHeight = element.style.lineHeight || defaultStyle.lineHeight;
+        let lineHeight = element.style.lineHeight || defaultStyle.lineHeight;
+        if (
+            !lineHeight &&
+            element.childNodes.length &&
+            element.childNodes.length === element.childElementCount
+        ) {
+            let childLineHeight = (<HTMLElement | null>(
+                element.children.item(0)
```

Comment:
Normally, you don't need to handle style from child node. They should be handled when element processor processes child node. If you see HTML like
```html
<div>
  <div style="line-height: ....">test</div>
</div>
```
Outer DIV doesn't have line-height, but that is ok, processor will create an empty paragraph for it but later it will be removed. Line height will be handled when inner DIV is being processed.

Reply:
And if you added some logic here, please also add unit test to cover it.

Reply:
Oh ok, if div only has one child with line-height, no need to process? I see
What about 
```html
<div style="...">
   <div style="line-height: ..."></div>
</div>
```
?

Usually clipboard html from word processors include the line height in a span under p, with both having styles

Reply:
I see. The line-height inside SPAN is a problem. Let me think about it.

Reply:
I decided to allow segments to hold a line-height, seems more straight forward to do here
Opinions on normalizing format after model is generated?